.cursor/rules/svelte-migration.mdc

```diff
@@ -19,6 +19,131 @@ Don't use createEventDispatcher for event handlers like on:click etc. Instead us
 - Check for global utilities imported from blade-core package specially theme utilities or theme types and constants and use those in svelte component. 
 - Ensure most of the styles are written using css classes. Component classes can be written in css files for that comonent's folder. Use Class-Variance-Authority for writing conditional css classes. Any global classes for styling props should go in common theme.css so that accessible by all components.
 
+
+## Blade Component Svelte Guidelines
+### Key Examples for references
+- Study the props of the React component which you are trying to migrate to Svelte. Props should remain consistent throughout Svelte and React components.
+
+## Directory Structure
+
+Components should be created in the following structure:
+
+```
+packages/
+└── blade-svelte/
+    └── src/
+        └── components/
+            ├── Button/
+            │   ├── Button.svelte
+            │   ├── types.ts
+            │   └── button.css
```

Comment:
where is cva config placed in this directory structure?

Reply:
oh wait the structure that is mentioned here seems different from the structure mentioned below in this file 🤔